compiler/extended/powprim.hh

```diff
@@ -146,16 +146,18 @@ class PowPrim : public xtended {
         if (isIntPowArg(types[1], *it, pow_arg)
             && (types[1]->variability() == kKonst)
             && (types[1]->computability() == kComp)
-            && (gGlobal->gNeedManualPow)) {
+            && (gGlobal->gNeedManualPow)
+            && (gGlobal->gOutputLang != "jax")) {
```

Comment:
What that?

Reply:
Setting `gGlobal->gNeedManualPow` to true/false is something each backend can decide in lib code.cpp, see for instance: https://github.com/grame-cncm/faust/blob/master-dev/compiler/libcode.cpp#L1435